src/test/java/com/wikia/webdriver/common/templates/NewTestTemplate_TwoDrivers.java

```diff
@@ -23,7 +23,7 @@
   public void start(Method method, Object[] data) {
     Configuration.clearCustomTestProperties();
     if (method.isAnnotationPresent(Execute.class)) {
-      if(!"".equals(method.getAnnotation(Execute.class).onWikia())){
+      if (!"".equals(method.getAnnotation(Execute.class).onWikia())) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Merge this if statement with the enclosing one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1066)